dspace/bin/make-handle-config

```diff
@@ -51,6 +51,7 @@ echo "Writing simple Handle server configuration"
 
 # Read parameters from DSpace config
 dshostname=`$BINDIR/dspace dsprop --property dspace.hostname`
+dshostip=`host $dshostname | awk '/has address/ { print $4; exit }'`
```

Comment:
This can make problems in an IPv4/IPv6 dual stack setup. Since version 7.2 the CNRI handle supports IPv6. That does not necessarily hinders us merging this, I just want to mention and discuss it.

BTW: Do we have a windows-version of this file? I'm surprised by every DSpace installations running on Windows, but there are some of them.